lib/module.js

```diff
@@ -506,6 +544,17 @@ Module.prototype.load = function(filename) {
   if (!Module._extensions[extension]) extension = '.js';
   Module._extensions[extension](this, filename);
   this.loaded = true;
+
```

Comment:
I think this should probably be gated by:

```js
if (experimentalModules) { }
```

so that without experimental modules enabled module behavior is as close to the old behavior as possible?

Reply:
fixed